src/otx/algorithms/common/adapters/mmcv/hooks/recording_forward_hook.py

```diff
@@ -238,3 +239,122 @@ def _get_mosaic_feature_map(self, feature_map: torch.Tensor, c: int, h: int, w:
                     mosaic_feature_map_mask[k, :, i, j] = torch.ones(c).to(feature_map.device)
             mosaic_feature_map = feature_map_repeated * mosaic_feature_map_mask
         return mosaic_feature_map
+
+
+class ViTReciproCAMHook(BaseRecordingForwardHook):
```

Comment:
Could you add comments to explain each argument of init function?

Reply:
Sure, added